src/components/avatar/Avatar.jsx

```diff
@@ -1,23 +1,52 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Icon, {TYPE as ICON_TYPE, ICON_COLOR} from '../icons/Icon';
+import Icon from '../icons/Icon';
 
-const DEFAULT_ICON = ICON_TYPE.PROFILE;
-const DEFAULT_COLOR = ICON_COLOR.GRAY_SECONDARY;
-const BORDER_SIZE = 2;
+export type AvatarSizeType =
+  | 'small'
+  | 'normal'
+  | 'large'
+  | 'xlarge'
+  | 'xxlarge';
 
 export const SIZE = {
-  SMALL: {className: 'small', iconSize: 24},
-  NORMAL: {className: 'normal', iconSize: 32},
-  LARGE: {className: 'large', iconSize: 48},
-  XLARGE: {className: 'xlarge', iconSize: 64},
-  XXLARGE: {className: 'xxlarge', iconSize: 120}
+  SMALL: 'small',
+  NORMAL: 'normal',
+  LARGE: 'large',
+  XLARGE: 'xlarge',
+  XXLARGE: 'xxlarge'
 };
 
-const Avatar = ({size = SIZE.NORMAL, border = false, spaced, imgSrc, className, link, title, ...props}) => {
+export const ICON_SIZE_PADDED = {
```

Comment:
@PinkaminaDianePie I am not sure about this namespace. Can we go with `ICON_SIZE_WITH_SPACE` or something similar?

Reply:
Or maybe somehow indicate that it is used for avatars with border?

Reply:
done